configuration/config.yml

```diff
@@ -15,5 +15,8 @@ analysers_settings:
     - requirements: "stats"
       scripts: 
```

Comment:
@VincenzoFerme When I unmarshall the YML file the document names are struct names, so if I do it like that I would need to make them fixed in the code.